builders/interact/substrate-api/chopsticks.md

```diff
@@ -0,0 +1,203 @@
+    If your browser cannot connect to the WebSocket endpoint provided by Chopsticks, you might need to allow insecure connections for the Polkadot.js Apps URL. Another solution is to run the [Docker version of Polkadot.js Apps](https://github.com/polkadot-js/apps#docker){target=\_blank}.
+
+## Replaying Blocks {: #replaying-blocks }
+
+If you would like to replay a block and retrieve its information to dissect the effects of an extrinsic, you can use the `yarn start run-block` command. Its following flags are:
+
+|            Flag            |                                      Description                                       |
+|:--------------------------:|:--------------------------------------------------------------------------------------:|
+|         `endpoint`         |                         The endpoint of the parachain to fork.                         |
+|          `block`           |            Use to specify at which block hash or number to replay the fork.            |
+|      `wasm-override`       |  Path of the WASM to use as the parachain runtime instead of an endpoint's runtime.   |
+|            `db`            |    Path to the name of the file that stores or will store the parachain's database.    |
+|          `config`          |                            Path or URL of the config file.                             |
+| `output-path=/[file_path]` |   Use to print out results to a JSON file instead of printing it out in the console.   |
+|           `html`           | Include to generate an HTML representation of the storage diff preview between blocks. |
+|           `open`           |                        Whether to open the HTML representation.                        |
+
+For example, running the following command will re-run the demo EVM ContainerChains's block 1000 and write the storage diff and other data in a `chain-output.json` file:  
+
+```bash
+yarn start run-block  \
+--endpoint wss://fraa-dancebox-3001-rpc.a.dancebox.tanssi.network  \
+--output-path=./chain-output.json  \
+--block 1000
+```
+
+## WebSocket Commands {: #websocket-commands }
+
+Chopstick's internal WebSocket server has special endpoints that allow the manipulation of the local Substrate chain. 
+
+These are the methods that can be invoked and their parameters:
+
+|      Method      |      Parameters       |                          Description                          |
+|:----------------:|:---------------------:|:-------------------------------------------------------------:|
+|  `dev_newBlock`  |       `options`       |               Generates one or more new blocks.               |
+| `dev_setStorage` | `values`, `blockHash` |         Create or overwrite the value of any storage.         |
+| `dev_timeTravel` |        `date`         |     Sets the timestamp of the block to the `date` value.      |
+|  `dev_setHead`   |    `hashOrNumber`     | Sets the head of the blockchain to a specific hash or number. |
+
+??? code "Parameters details"
```

Comment:
I love a good expandable code snippet moment! haha

Reply:
After continuing my review, I feel like the initial problem wasn't addressed. So if you want to implement an expandable snippet for each item (like we have in [Moonbeam docs](https://docs.moonbeam.network/builders/interoperability/xcm/xc20/send-xc20s/xtokens-pallet/#extrinsics)), you can. Or if there aren't too many tabbed items spamming the screen, you don't have to use expandable snippets